cmd/rookflex/cmd/mount.go

```diff
@@ -174,3 +181,63 @@ func mount(client *rpc.Client, mounter *k8smount.SafeFormatAndMount, globalVolum
 	}
 	return err
 }
+
+func mountCephFS(client *rpc.Client, opts *flexvolume.AttachOptions) error {
+
+	log(client, fmt.Sprintf("mounting ceph filesystem %s on %s", opts.VolumeName, opts.MountDir), false)
+
+	if opts.ClusterName == "" {
+		return fmt.Errorf("Rook: Attach filesystem %s failed: cluster is not provided", opts.VolumeName)
+	}
+
+	// Get client access info
+	var clientAccessInfo model.ClientAccessInfo
+	err := client.Call("FlexvolumeController.GetClientAccessInfo", opts.ClusterName, &clientAccessInfo)
+	if err != nil {
+		log(client, fmt.Sprintf("Attach filesystem %s on cluster %s failed: %v", opts.VolumeName, opts.ClusterName, err), true)
+		return fmt.Errorf("Rook: Attach filesystem %s on cluster %s failed: %v", opts.VolumeName, opts.ClusterName, err)
+	}
+
+	path := string(os.PathSeparator)
+	if opts.Path != "" {
+		path = opts.Path
+		if !strings.HasPrefix(path, string(os.PathSeparator)) {
+			path = string(os.PathSeparator) + path
+		}
+	}
+
+	devicePath := fmt.Sprintf("%s:%s", strings.Join(clientAccessInfo.MonAddresses, ","), path)
+	options := []string{fmt.Sprintf("name=%s", clientAccessInfo.UserName), fmt.Sprintf("secret=%s", clientAccessInfo.SecretKey), fmt.Sprintf("mds_namespace=%s", opts.VolumeName)}
+
+	log(client, fmt.Sprintf("mounting ceph filesystem %s on %s to %s", opts.VolumeName, devicePath, opts.MountDir), false)
+	mounter := getMounter()
+	err = redirectStdout(
+		client,
+		func() error {
+
+			notMnt, err := mounter.Interface.IsLikelyNotMountPoint(opts.MountDir)
+			if err != nil && !os.IsNotExist(err) {
+				return err
+			}
+			if !notMnt {
+				return nil
+			}
+			os.MkdirAll(opts.MountDir, 0750)
+
+			err = mounter.Interface.Mount(devicePath, opts.MountDir, "ceph", options)
```

Comment:
here's another place where the "ceph" filesystem type string could be made a const

Reply:
ok